controllers/networkchaos/trafficcontrol/types.go

```diff
@@ -283,8 +301,12 @@ func (r *endpoint) applyTc(ctx context.Context, sources, targets []v1.Pod, exter
 		return nil
 	}
 
+	if len(ipSetPostFix) == 0 {
+		ipSetPostFix = string(tcType[0:5])
+	}
+
 	// create ipset contains all target ips
-	dstIpset := ipset.BuildIPSet(targets, externalTargets, networkchaos, string(tcType)[0:5], m.Source)
+	dstIpset := ipset.BuildIPSet(targets, externalTargets, networkchaos, ipSetPostFix, m.Source)
```

Comment:
How about that:

```go
dstIpset := ipset.BuildIPSet(targets, externalTargets, networkchaos, string(tcType)[0:5] + ipSetPostFix, m.Source)
```

It avoids the modification for `ipSetPostFix`, and still shows the `tcType` in the ipset name whether `ipSetPostFix` is empty or not.

Reply:
Maybe we could also shrink the length of `string(tcType)[0:5]`, as I know the only two possible `TcType` is `netem` and  `bandwidth`, and they are easy to distinguish

Reply:
`string(tcType)[0:2]` ?